batch/batch/cloud/gcp/instance_config.py

```diff
@@ -49,15 +52,28 @@ def create(
         assert machine_type_parts is not None, machine_type
         instance_family = machine_type_parts.machine_family
 
-        resources = [
-            GCPComputeResource.create(product_versions, instance_family, preemptible, region),
-            GCPMemoryResource.create(product_versions, instance_family, preemptible, region),
-            GCPStaticSizedDiskResource.create(product_versions, 'pd-ssd', boot_disk_size_gb, region),
-            data_disk_resource,
-            GCPDynamicSizedDiskResource.create(product_versions, 'pd-ssd', region),
-            GCPIPFeeResource.create(product_versions, 1024),
-            GCPServiceFeeResource.create(product_versions),
-        ]
+        if GPU_VERSIONS.get(machine_type_parts.machine_family):
```

Comment:
Better not to duplicate the resources list.  Just use `append` to add the accelerator resource to the list of resources if an accelerator is present:

```
  resources = [...]

  accelerator_family = GPU_VERSIONS.get(machine_type_parts.machine_family)
  if acclerator_family:
    resources.append(GCPAccleratorResource.create(...))
```